src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/ChangeOfPriceProviderControllerTests.cs

```diff
@@ -129,37 +128,37 @@ public async Task ProviderInitiatedCheckDetailsPage_InvalidModel_ReturnsProvider
             viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderEnterChangeDetailsViewName);
         }
 
-		[Test]
-		public async Task ProviderInitiatedCheckDetailsPage_ReturnsProviderInitiatedCheckDetailsViewName()
-		{
-			// Arrange
-			var createChangeOfPriceModel = _fixture.Create<ProviderChangeOfPriceModel>();
-			var controller = new ChangeOfPriceProviderController(_mockLogger.Object, _mockApprenticeshipService.Object, _mockMapper.Object, _mockCacheService.Object, _mockExternalUrlHelper.Object);
+        [Test]
+        public async Task ProviderCheckDetails_ReturnsProviderInitiatedCheckDetailsViewName()
+        {
+            // Arrange
+            var createChangeOfPriceModel = _fixture.Create<ProviderChangeOfPriceModel>();
+            var controller = new ChangeOfPriceProviderController(_mockLogger.Object, _mockApprenticeshipService.Object, _mockMapper.Object, _mockCacheService.Object, _mockExternalUrlHelper.Object);
             controller.SetupHttpContext(_fixture.Create<long>(), "anyApprenticeshipId");
 
             // Act
             var result = await controller.ProviderCheckDetails(createChangeOfPriceModel);
 
-			// Assert
-			var viewResult = result.ShouldBeOfType<ViewResult>();
-			viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderCheckDetailsViewName);
-		}
+            // Assert
+            var viewResult = result.ShouldBeOfType<ViewResult>();
+            viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderCheckDetailsViewName);
+        }
 
-		[Test]
-		public void GetProviderInitiatedEditPage_ReturnsProviderInitiatedViewName()
-		{
-			// Arrange
-			var createChangeOfPriceModel = _fixture.Create<ProviderChangeOfPriceModel>();
-			var controller = new ChangeOfPriceProviderController(_mockLogger.Object, _mockApprenticeshipService.Object, _mockMapper.Object, _mockCacheService.Object, _mockExternalUrlHelper.Object);
+        [Test]
+        public void GetProviderInitiatedEditPage_ReturnsProviderInitiatedViewName()
+        {
+            // Arrange
+            var createChangeOfPriceModel = _fixture.Create<ProviderChangeOfPriceModel>();
+            var controller = new ChangeOfPriceProviderController(_mockLogger.Object, _mockApprenticeshipService.Object, _mockMapper.Object, _mockCacheService.Object, _mockExternalUrlHelper.Object);
 
-			// Act
-			var result = controller.GetProviderEditChangeDetails(createChangeOfPriceModel);
+            // Act
+            var result = controller.GetProviderEditChangeDetails(createChangeOfPriceModel);
 
-			// Assert
-			var viewResult = result.ShouldBeOfType<ViewResult>();
-			viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderEnterChangeDetailsViewName);
+            // Assert
+            var viewResult = result.ShouldBeOfType<ViewResult>();
+            viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderEnterChangeDetailsViewName);
         }
-        
+
```

Comment:
I'm assuming all this tabbing looks right in the IDE?

Reply:
Yes, it was all over the place